modules/ocf/files/packages/cups/raster-filter

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# A Tea4CUPS filter to perform rasterization.
+# Based off the pdf-open script
+
+set -euo pipefail
+
+input=$(mktemp --suffix='.pdf')
+
+# Feed filter input into temporary file
+cat - > "$input"
+
+# If the file is not a PDF, then just pass it thorugh
+mimetype="$(file -bi "$input")"
+
+if [ "${mimetype%%;*}" != "application/pdf" ]; then
+    cat "$input"
+    exit
+fi
+
+# Rasterizing PDF...
+output=$(mktemp --suffix='.pdf')
+error=$(mktemp)
+
+if ! convert -density 250 "$input" "$output" 2> "$error"; then
```

Comment:
You are using two distinct styles for your if statements in this script. Please stick to one.

Reply:
Addressed... I've removed the `-e` option when setting Bash settings, and now I check the return status of `convert`... see commit https://github.com/ocf/puppet/pull/684/commits/6bfcfc5c0b0dbbf0123dcaa5752ce5afa2605534